openslides/agenda/serializers.py

```diff
@@ -56,11 +52,7 @@ class Meta:
         fields = (
             'id',
             'item_number',
-            'item_no',
             'title',
```

Comment:
Good point. But it is handled correctly by the django rest framework:
(From the option request)
```
...
            "agenda_item_id": {
                "type": "field",
                "required": false,
                "read_only": true,
                "label": "Agenda item id"
            }
...
```